docs/datamodel/scalars/numeric.rst

```diff
@@ -103,6 +103,16 @@ from :eql:type:`str` and :eql:type:`json`.
         db> SELECT 42n IS bigint;
         {true}
 
+    To represent really big integers it is possible to even use the
+    exponent notation (e.g. ``1e20`` instead of
+    ``100000000000000000000``) as long as the exponent is positive and
+    there is not dot in the mantissa.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT 1e+100n IS bigint;
+        {true}
+
     Note that a float literal followed by ‘n’ produces a
```

Comment:
Hm, while we're here: it's not a float literal, it's a bigint literal. The language is confusing (even though I see why it's written that way).

Reply:
I've reworded to make a bit clearer.